src/metabase/query_processor/middleware/constraints.clj

```diff
@@ -34,9 +34,7 @@
   :type           :integer
   :database-local :allowed
   :audit          :getter
-  :doc "Must be less than 1048575, and less than the number configured in MB_AGGREGATED_QUERY_ROW_LIMIT.
-        This environment variable also affects how many rows Metabase returns in dashboard subscription attachments.
-        See also MB_AGGREGATED_QUERY_ROW_LIMIT.")
+  :doc "Must be less than 1048575, and less than the number configured in MB_AGGREGATED_QUERY_ROW_LIMIT. See also MB_AGGREGATED_QUERY_ROW_LIMIT.")
```

Comment:
Huh, apparently we have a test that verifies formatting for this var, hence the test failures. Let me know if you want help updating the test; otherwise, I think you can just edit this guy to match the changes here and the tests should pass. OTOH, maybe this var was specifically chosen because it contained newlines?

https://github.com/metabase/metabase/blob/e104ed81e5bdcd70d7983e179d524100015c5760/test/metabase/cmd/env_var_dox_test.clj#L16